src/bitcoin-cli.cpp

```diff
@@ -151,10 +151,15 @@ static int AppInitRPC(int argc, char* argv[])
             strUsage += FormatParagraph(LicenseInfo());
         } else {
             strUsage += "\n"
-                "Usage:  bitcoin-cli [options] <command> [params]  Send command to " PACKAGE_NAME "\n"
-                "or:     bitcoin-cli [options] -named <command> [name=value]...  Send command to " PACKAGE_NAME " (with named arguments)\n"
-                "or:     bitcoin-cli [options] help                List commands\n"
-                "or:     bitcoin-cli [options] help <command>      Get help for a command\n";
```

Comment:
I think we might be losing some clarity by splitting the descriptions  up from the usage lines

Reply:
Yeah I did think about this, but I did add the same in formation to the paragraph above. Also, the current format is somewhat "broken" when geenrating manpages currently (see first image in OP).

I also checked a load of man pages and `--help` docs from other popular utils, and it seems that our current format is the outlier, in terms of including explanatory text inline with examples/usage strings.

Reply:
Yeah I did think about this, but I did add the same information to the paragraph above. Also, the current format is somewhat "broken" when genrating manpages currently (see first image in OP, all sorts of messy underlines).

I also checked a load of man pages and `--help` docs from other popular utils, and it seems that our current format is the outlier, in terms of including explanatory text inline with examples/usage strings.